### ISSUE TITLE
No connection between vessel and relays

### ISSUE THREAD

**Issue by codebori** [DRVeyl/RealAntennas#37]
I am not sure if I just don't understand how it works or if this is a bug, but relaying is not always working for me.

I have a vessel in a 200km orbit around earth with an omni antenna that can transmit and receive on S-band at 288.6 Kbps at 100Mm. https://i.imgur.com/XL2XY5r.png

I have 4 relay satellites in a geostationary orbit (35793km) with a dish pointed at earth that can transmit and receive on S-band at 288.6 Kbps at 100Mm. https://i.imgur.com/UdepFHb.png

The relay satellites cover earth completely: https://i.imgur.com/1XdaI37.png

Still the vessel is regularly disconnected (from earth based dsn's) and never connects through the relays. Why is that?

**Comment by DRVeyl** [comment]
Most likely, you are using the planner to check the different antennas individually against one of the 3 large DSN stations [with their very large, high-gain dishes] and not against eachother.

<https://github.com/DRVeyl/RealAntennas/wiki/Antenna-Planning-Guide> is a wiki article on using the in-game planner.  The second example, half-way down the article, illustrates how to plan a connection between two craft.  There are a couple extra steps to tell the planner to plan with a particular ground station or a particular antenna on another vessel.

Looking over the pics, the good news for you is the relay antennas likely have enough transmit power already, and it's only the vessel in LEO that needs some extra power.

**Comment by codebori** [comment]
I am so sorry, that example is exactly my situation. I did read everything at my first install, I didn't really get that part, but I should have revisited it before posting this. 
Thank you very much for your patience. I myself cannot stand it when people ask questions that have already been answered in the documentation. 
(Can I just leave this open until after I have actually tried it? I'm sure it will work now, but just in case.)
